### DISCUSSION TITLE
S3 Postman API Sample

### DISCUSSION THREAD

**Discussion by rezaeclipse88tech** [awsugid/forum#8] [Q&A]
Adakah AWS S3 Postman API untuk upload file dan melihat file ?

**Comment by bellabeen** [comment]
Mungkin bisa dicoba cek di dokumentasi berikut https://repost.aws/questions/QUxmLkwTIcSQiW5qatM4GRIQ/trying-to-access-s3-bucket-using-postman-app-and-fix-accessdenied-error

**Reply by rezaeclipse88tech** [reply]
Makasih, so far sudah dapat

**Comment by goFrendiAsgard** [comment]
Barangkali tertarik untuk akses S3 secara programmatically, bisa pakai boto3.
Nanti bisa juga bikin web app yang serve REST API versi kita.

https://felipeagq99.medium.com/connect-to-a-aws-s3-bucket-with-boto3-of-python-6f22441c5390

**Reply by rezaeclipse88tech** [reply]
Makasih .. cuma pengen coba2 tidak pakai coding dulu

**Comment by rezaeclipse88tech** [comment]
Saya sudah nemu postman linknya, setelah melalang buana via GOOGLE cukup banyak contohnya
